docs/outlook/online-meeting.md

```diff
@@ -0,0 +1,182 @@
+
+> [!NOTE]
+> This feature is only supported in [preview](../reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) on Android with an Office 365 subscription.
+
+## Scenario: Online-meeting provider
+
+In this section, you'll learn how to set up your Outlook mobile add-in to enable users to organize and join a meeting using your online-meeting service. Throughout this article, we'll be using a fictional online-meeting service provider, "Contoso".
+
+### Configure the manifest
+
+To enable users to create online meetings with your add-in, you must configure the `MobileOnlineMeetingCommandSurface` extension point in the manifest under the parent element `MobileFormFactor`. Other form factors are not supported.
+
+The following example shows a sample of the manifest that includes the `MobileFormFactor` element and `MobileOnlineMeetingCommandSurface` extension point.
+
+```xml
+...
+<VersionOverrides xmlns="http://schemas.microsoft.com/office/mailappversionoverrides" xsi:type="VersionOverridesV1_0">
+  <VersionOverrides xmlns="http://schemas.microsoft.com/office/mailappversionoverrides/1.1" xsi:type="VersionOverridesV1_1">
+    ...
+    <Hosts>
+      <Host xsi:type="MailHost">
+        <MobileFormFactor>
+          <FunctionFile resid="residMobileFuncUrl" />
+          <ExtensionPoint xsi:type="MobileOnlineMeetingCommandSurface">
+            <!-- Configure selected extension point. -->
+            <Control xsi:type="MobileButton" id="onlineMeetingFunctionButton">
+              <Label resid="residUILessButton0Name" />
+              <Icon>
+                <bt:Image resid="UiLessIcon" size="25" scale="1" />
+                <bt:Image resid="UiLessIcon" size="25" scale="2" />
+                <bt:Image resid="UiLessIcon" size="25" scale="3" />
+                <bt:Image resid="UiLessIcon" size="32" scale="1" />
+                <bt:Image resid="UiLessIcon" size="32" scale="2" />
+                <bt:Image resid="UiLessIcon" size="32" scale="2" />
+                <bt:Image resid="UiLessIcon" size="48" scale="1" />
+                <bt:Image resid="UiLessIcon" size="48" scale="2" />
+                <bt:Image resid="UiLessIcon" size="48" scale="3" />
+              </Icon>
+              <Action xsi:type="ExecuteFunction">
+                <FunctionName>insertContosoMeeting</FunctionName>
+              </Action>
+            </Control>
+          </ExtensionPoint>
+        </MobileFormFactor>
+      </Host>
+    </Hosts>
+    ...
+  </VersionOverrides>
+</VersionOverrides>
+...
+```
+
+### Implementation
+
+In this section, you'll see an outline for how your add-in script can update a user's meeting to include online meeting details.
+
+The following example shows how you could construct online meeting details. Not shown is how to get the meeting organizer's ID and other details from your service.
```

Comment:
@lindalu-MSFT My thinking here was that call that out since we're not even calling a fake service, which would have at least provided calls to get data from the service. Service providers should know (or determine otherwise) how to call into their own service.